pkg/utils/release/controller.go

```diff
@@ -39,50 +44,40 @@ func (s *SuiteController) CreateApplicationSnapshot(name string, namespace strin
 }
 
 // CreateRelease creates a new Release using the given parameters.
-func (s *SuiteController) CreateRelease(name, namespace, snapshot, sourceReleaseLink string) (*v1alpha1.Release, error) {
-	release := &v1alpha1.Release{
+func (s *SuiteController) CreateRelease(name, source_namespace, snapshot, sourceReleasePlan string) (*appstudiov1alpha1.Release, error) {
+	release := &appstudiov1alpha1.Release{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
-			Namespace: namespace,
+			Namespace: source_namespace,
 		},
 		Spec: v1alpha1.ReleaseSpec{
 			ApplicationSnapshot: snapshot,
-			ReleaseLink:         sourceReleaseLink,
+			ReleasePlan:         sourceReleasePlan,
 		},
 	}
 
 	return release, s.KubeRest().Create(context.TODO(), release)
 }
 
-// CreateReleaseLink creates a new ReleaseLink using the given parameters.
-func (s *SuiteController) CreateReleaseLink(name, namespace, application, targetNamespace, releaseStrategy string) (*v1alpha1.ReleaseLink, error) {
-	releaseLink := &v1alpha1.ReleaseLink{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      name,
-			Namespace: namespace,
-		},
-		Spec: v1alpha1.ReleaseLinkSpec{
-			DisplayName:     name,
-			Application:     application,
-			Target:          targetNamespace,
-			ReleaseStrategy: releaseStrategy,
-		},
-	}
-
-	return releaseLink, s.KubeRest().Create(context.TODO(), releaseLink)
-}
-
 // CreateReleaseStrategy creates a new ReleaseStrategy using the given parameters.
-func (s *SuiteController) CreateReleaseStrategy(name, namespace, pipelineName, bundle string, policy string) (*v1alpha1.ReleaseStrategy, error) {
-	releaseStrategy := &v1alpha1.ReleaseStrategy{
+func (s *SuiteController) CreateReleaseStrategy(name, target_namespace, pipelineName, bundle string, policy string, service_account string) (*appstudiov1alpha1.ReleaseStrategy, error) {
+	releaseStrategy := &appstudiov1alpha1.ReleaseStrategy{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
-			Namespace: namespace,
+			Namespace: target_namespace,
 		},
 		Spec: v1alpha1.ReleaseStrategySpec{
 			Pipeline: pipelineName,
 			Bundle:   bundle,
 			Policy:   policy,
+			Params: []appstudiov1alpha1.Params{
```

Comment:
This is not fixing anything. This is adding functionality and you are making the tests to depend on a repository that we do not control.